packages/nimble-components/src/anchor-tree-item/index.ts

```diff
@@ -0,0 +1,136 @@
+import { attr } from '@microsoft/fast-element';
+import {
+    DesignSystem,
+    AnchorOptions,
+    StartEnd,
+    applyMixins,
+    isTreeItemElement,
+    TreeItem as FoundationTreeItem
+} from '@microsoft/fast-foundation';
+import { keyArrowLeft, keyEnter } from '@microsoft/fast-web-utilities';
+import { AnchorBase } from '../anchor-base';
+import { styles } from './styles';
+import { template } from './template';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-anchor-tree-item': AnchorTreeItem;
+    }
+}
+
+/**
+ * A nimble-styled anchor tree item
+ */
+export class AnchorTreeItem extends AnchorBase {
+    /**
+     * When true, the control will appear selected by user interaction.
+     * @public
+     * @remarks
+     * HTML Attribute: selected
+     */
+    @attr({ mode: 'boolean' })
+    public selected = false;
+
+    /**
+     * When true, the control will be immutable by user interaction. See {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/disabled | disabled HTML attribute} for more information.
+     * @public
+     * @remarks
+     * HTML Attribute: disabled
+     */
+    @attr({ mode: 'boolean' })
+    public disabled = false;
+
+    /**
+     * Whether the tree is nested
+     *
+     * @public
+     */
+    public isNestedItem(): boolean {
+        return isTreeItemElement(this.parentElement as Element);
+    }
+
+    /**
+     * Handle focus events
+     *
+     * @internal
+     */
+    public handleFocus(_e: FocusEvent): void {
+        this.setAttribute('tabindex', '0');
+    }
+
+    /**
+     * Handle blur events
+     *
+     * @internal
+     */
+    public handleBlur(_e: FocusEvent): void {
+        this.setAttribute('tabindex', '-1');
+    }
+
+    /**
+     * @internal
+     */
+    public keydownHandler(e: KeyboardEvent): boolean {
+        if (e.defaultPrevented) {
+            return false;
+        }
+        switch (e.key) {
+            case keyEnter:
+                // Do not let the event bubble up to the FAST tree, or it will
+                // prevent the default action.
+                e.stopPropagation();
+                break;
+            case keyArrowLeft:
+                // For FAST tree items, the FAST tree view handles this navigation,
+                // but since our anchor tree item is not "instanceof FASTTreeItem",
+                // the FAST tree view won't do this for us. We do it ourselves.
+                if (this.parentElement && this.isNestedItem()) {
+                    FoundationTreeItem.focusItem(this.parentElement);
+                }
+                break;
+            default:
+        }
+        return true;
+    }
+
+    /**
+     * Apparently, pressing the Enter key on an anchor generates a click event, too.
```

Comment:
Yea I get that keyboard events resulting in a click might be unexpected, but it's an accessibility thing: https://html.spec.whatwg.org/#activation

> The user agent should allow the user to manually trigger elements that have an [activation behavior](https://dom.spec.whatwg.org/#eventtarget-activation-behavior), for instance using keyboard or voice input, or through mouse clicks. When the user triggers an element with a defined [activation behavior](https://dom.spec.whatwg.org/#eventtarget-activation-behavior) in a manner other than clicking it, the default action of the interaction event must be to [fire a click event](https://html.spec.whatwg.org/#fire-a-click-event) at the element.

It's similar to the idea of calling it dismissing a dialog instead of listening only to the escape key. The user agent can dismiss a dialog multiple ways (and represents it by a `cancel` event) and it can trigger the activation of an element multiple ways (and represents it by a maybe poorly named / historical `click` event). 

Any controls that have a default behavior associated with them will fire that click event to signify that activation.

```suggestion
     * Activating the anchor by pressing the Enter key results in a click event.
```

Reply:
Made this change myself.